hoomd/mpcd/pytest/test_stream.py

```diff
@@ -65,6 +65,58 @@ def test_pickling(self, simulation_factory, snap, cls, init_args):
         sim.run(0)
         pickling_check(sm)
 
+    @pytest.mark.parametrize(
+        "force",
+        [
+            hoomd.mpcd.force.BlockForce(
+                force=2.0, half_separation=3.0, half_width=0.5),
+            hoomd.mpcd.force.ConstantForce(force=(1, -2, 3)),
+            hoomd.mpcd.force.SineForce(amplitude=2.0, wavenumber=1),
```

Comment:
Should a case where no force is added to the streaming method be included for completeness?

Reply:
I think this case is covered by `test_pickling` because `solvent_force=None` is the default, but it doesn't hurt to explicitly test with `None` here as well. Added!